compiler/dyno/include/chpl/resolution/scope-queries.h

```diff
@@ -43,13 +43,10 @@ namespace resolution {
   const Scope* scopeForModule(Context* context, ID moduleId);
 
   /**
-    Given a Scope, compute the ResolvedVisibilityScope
-    by processing the use/import statements in order.
-
-    If the scope didn't have use/imports, returns nullptr.
+    Gather the IDs of Use/Import statements within a scope.
    */
-  const ResolvedVisibilityScope* resolveVisibilityStmts(Context* context,
-                                                        const Scope* scope);
+  const std::vector<ID>& findUseImportStmts(Context* context,
```

Comment:
No because it's returning just the IDs of the use/import statements which is not the same as a "visibility path" in terms of concepts. It is true that the current implementation of VisibilityPath happens to be the same.